var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -46,10 +46,13 @@ class Boost(Package):
             branch='develop',
             submodules=True)
 
+    version('1.67.0', '694ae3f4f899d1a80eb7a3b31b33be73c423c1ae',
+            url='https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2',
+            preferred=True)
     version('1.67.0.b1', '3423a4a3ec5297051fd27091864630e2dce4c159',
```

Comment:
Now that `1.67.0` has been officially released, can we remove this beta version? Then we can remove `preferred=True` as well.

Reply:
I agree, we can remove `beta` and its patch.